docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+web:
+  build: .
+  links:
+   - db
+  ports:
+   - "8080:8000"
```

Comment:
Here the port should be exposed at `8000` instead of `8080` or was it intentional?